packages/react-sdk/src/components/text-block.tsx

```diff
@@ -1,10 +1,19 @@
-import { forwardRef, type ElementRef, type ComponentProps } from "react";
+import {
+  createElement,
+  forwardRef,
+  type ElementRef,
+  type ComponentProps,
+} from "react";
 
 export const defaultTag = "div";
 
-export const TextBlock = forwardRef<
-  ElementRef<typeof defaultTag>,
-  ComponentProps<typeof defaultTag>
->((props, ref) => <div {...props} ref={ref} />);
+// We don't want to enable all tags because Box is usually a container and we have specific components for many tags.
+type Props = ComponentProps<typeof defaultTag> & {
+  tag?: "div" | "span" | "figcaption";
```

Comment:
Span is not block

Reply:
Yes, which is why we need to rename to Text, we already discussed it in another PR

Reply:
Sometimes you need a span

Reply:
e.g. when text is inside a label, you don't want a div https://github.com/webstudio-is/webstudio-builder/issues/1552

Reply:
Label is rich text though

Reply:
Let's discuss in dm if its a problem, merging now